ui/components/DashboardComponent/resources/resources-sub-menu.js

```diff
@@ -29,6 +29,9 @@ const styles = (theme) => ({
     '& .MuiTabs-indicator': {
       backgroundColor: theme.palette.type === 'dark' ? '#00B39F' : theme.palette.primary,
     },
+    '& .MuiTabs-flexContainer': {
+      justifyContent: 'space-between',
```

Comment:
```suggestion
      justifyContent: 'center',
```
@Aniket-76 I think this will look good.

Reply:
by using this, some other problems are arising in some tabs